contracts/src/v0.8/keystone/CapabilityRegistry.sol

```diff
@@ -559,6 +567,20 @@ contract CapabilityRegistry is OwnerIsCreator, TypeAndVersionInterface {
     emit DONAdded(id, isPublic);
   }
 
+  /// @notice Removes DONs from the Capability Registry
+  /// @param donIds The IDs of the DON to be removed
+  function removeDONs(uint32[] calldata donIds) external onlyOwner {
```

Comment:
We should probably update `addDON` to `addDONs` later in a followup PR

Reply:
I feel like it would make that function extremely complex.